lib/Backend/GlobOptBailOut.cpp

```diff
@@ -951,7 +957,7 @@ GlobOpt::FillBailOutInfo(BasicBlock *block, BailOutInfo * bailOutInfo)
                 {
                     sym = opnd->GetStackSym();
                     Assert(FindValue(sym));
-                    this->blockData.capturedArgs->Set(sym->m_id);
+                    this->blockData.changedSyms->Set(sym->m_id);
```

Comment:
Args need to be captured even if without setting, will add a comment.

---
In reply to: [71631271](https://github.com/Microsoft/ChakraCore/pull/1247#discussion_r71631271) [](ancestors = 71631271)